cmd/adapter/main.go

```diff
@@ -175,12 +175,12 @@ func printWelcomeMsg(cmd *Adapter) error {
 		logger.Error(err, "not able to get Kubernetes version")
 		return err
 	}
-	version, err := clientset.ServerVersion()
+	k8sVersion, err := clientset.ServerVersion()
 	if err != nil {
 		logger.Error(err, "not able to get Kubernetes version")
 		return err
 	}
-	kedautil.PrintWelcome(logger, kedautil.NewK8sVersion(version), "metrics server")
+	kedautil.PrintWelcome(logger, kedautil.NewK8sVersion(k8sVersion), "metrics server")
```

Comment:
this change is no longer needed, could you please skip it?

Reply:
Done